### ISSUE TITLE
Data is missing

### ISSUE THREAD

**Issue by kousha789123** [eliracho37/Donation-Based-Crowdfunding-Data#4]
Can you provide this data again? my contact: www.kshafi.com

**Comment by aprajnaparamita** [comment]
yeah, I was very interested in this project and the methods used. I'm doing a similar personal project but don't understand the deeper math to predict this.
